setup.py

```diff
@@ -44,4 +44,6 @@
     ],
     keywords='kaitai,struct,construct,ksy,declarative,data structure,data format,file format,packet format,binary,parser,parsing,unpack,development',
     py_modules=["kaitaistruct"],
+    requires=["RecursionSafe"],
+    dependency_links=["git+https://github.com/KOLANICH/RecursionSafe.py.git"],
```

Comment:
i wouldn't trust a python library with camelcase naming and `.py` suffix. ;-)

Reply:
Also:

> Requirements
> Python 3.

Which is, obviously, a deal-breaker for inclusion here anyway :(

> Python 2 is dead, stop raping its corpse.

Unfortunately, I see the exact opposite :(